src/components/MainView.vue

```diff
@@ -1,40 +1,47 @@
 <template>
-	<div class="view-wrapper">
-		<div class="header-wrapper bordered-container">
-		</div>
-		<div class="body-wrapper bordered-container">
-			<div class="left-nav-wrapper bordered-container"></div><div class="body-content-wrapper bordered-container">
-				<recon-console-toolbar
-					v-on:displayFilterDialog_click="displayFilterDialog2"
-					/>
+	<div class="view-resize-wrapper" v-resize="onResize">
+		<div class="view-wrapper">
+			<div class="header-wrapper bordered-container">
 			</div>
+			<div class="body-wrapper bordered-container">
+				<div class="left-nav-wrapper bordered-container"></div><div class="body-content-wrapper bordered-container">
+					<recon-console-toolbar
+						v-on:displayFilterDialog_click="displayFilterDialog2"
+						/>
+				</div>
+			</div>
+			<div class="footer-wrapper bordered-container"></div>
+			<transition name="fade">
+			<keep-alive>
+				<filter-dialog
+					v-bind:filterList="filterList"
+					v-bind:accessLevelList="accessLevelList"
+					v-if="displayFilterDialog"
+					v-on:buttonClick="filterDialogButton_click"
+					/>
+			</keep-alive>
+			</transition>
 		</div>
-		<div class="footer-wrapper bordered-container"></div>
-		<transition name="fade">
-		<keep-alive>
-			<filter-dialog
-				v-bind:filterList="filterList"
-				v-bind:accessLevelList="accessLevelList"
-				v-if="displayFilterDialog"
-				v-on:buttonClick="filterDialogButton_click"
-				/>
-		</keep-alive>
-		</transition>
 	</div>
 </template>
 
 <script>
 import ReconConsoleToolbar from "./ReconConsoleToolbar";
 import FilterDialog from "./FilterDialog";
+//var resize = require("vue-resize-directive");
+import resize from 'vue-resize-directive';
 
 export default {
 	components: {ReconConsoleToolbar, FilterDialog},
+	directives: {
+		resize
+	},
 	mounted: function() {
-		window.addEventListener('resize.', this.fitToContainer.bind(this, this.$el));
+		//window.addEventListener('resize.', this.fitToContainer.bind(this, this.$el));
 		//this.fitToContainer(this.$el);
 	},
 	beforeDestroy: function() {
-		window.removeEventListener('resize', this.fitToContainer);
+		//window.removeEventListener('resize', this.fitToContainer);
 	},
 	props: [ 'filterList', 'accessLevelList' ],
 	data: function() {
```

Comment:
This would be a good place for arrow-function shorthand. :)
```
data: () => ({ displayFilterDialog: false })
```
If the method body has nothing but a `return` statement in it - you can simply put the return value on the right side of the `=>`, with no enclosing body or `return` keyword. In this case, since you're returning an object literal, you need to enclose it with `()`. This is just so the transpiler understands that the `{}`'s indicate an object literal, rather than a method body.

Reply:
👍